frontend/tasks/content.js

```diff
@@ -196,6 +245,7 @@ function extractNewsUpdatesFromExperiment(experiment) {
   return news_updates.map(update => ({
     ...update, experimentSlug: slug
   }));
+  delete experiment.news_updates;
```

Comment:
This line will never be reached, since the previous line is `return`. I'd had this in this function earlier, but had to [move it into the gulp tasks](https://github.com/mozilla/testpilot/commit/cc33d43006c9b5b53e9e84a65028c89ee89c66a9#diff-52fa3e5c3e785506b7cc22f3df8cc3fbR47) because modifying these parameters in place ended up not working after Babel ran over the code.